### PR TITLE
feat: Misc alignments

### PR DESCRIPTION
**Pull request by ebariche**

### CODE REVIEW

**Review by davidjohnoliver — COMMENTED** [review]
You've changed the generation logic - where are the resulting changes to the generated files?

File: src/Uno.UWPSyncGenerator/Generator.cs [hunk 1]
```diff
@@ -77,6 +78,7 @@ public virtual void Build(string basePath, string baseName, string sourceAssembl
 
 			_voidSymbol = _referenceCompilation.GetTypeByMetadataName("System.Void");
 			_dependencyPropertySymbol = _referenceCompilation.GetTypeByMetadataName(BaseXamlNamespace + ".DependencyProperty");
+			FlagsAttributeSymbol = _referenceCompilation.GetTypeByMetadataName("System.FlagsAttribute");
```
Comment (jeromelaban) [inline src/Uno.UWPSyncGenerator/Generator.cs:81]:
```suggestion
			_flagsAttributeSymbol = _referenceCompilation.GetTypeByMetadataName("System.FlagsAttribute");
```

**Pull request merged**